unumpy/_multimethods.py

```diff
@@ -322,6 +322,17 @@ def accumulate(self, a, axis=0, dtype=None, out=None):
 trunc = ufunc("trunc", 1, 1)
 
 
+@create_numpy(_dtype_argreplacer)
+def empty(shape, dtype=float, order='C'):
+    return (mark_dtype(dtype),)
+
+
+@create_numpy(_dtype_argreplacer)
+@all_of_type(ndarray)
+def empty_like(prototype, dtype=None, order='K', subok=True, shape=None):
+    return (mark_dtype(dtype),)
```

Comment:
Prototype needs to be marked as an `ndarray`. The `all_of_ndarray` decorator takes care of that, so this line should be simply:
```suggestion
    return prototype, mark_dtype(dtype)
```

Reply:
Is there a difference between your suggestion and the one below?

```suggestion
    return (prototype, mark_dtype(dtype),)
```

I'm asking since other multimethods seem to conform to this type of writing.

Thank you!

Reply:
Not really, they both compile down to the same bytecode. When it's a single element, we put a comma after it to denote it's a `tuple`. When multiple, we don't put the trailing comma. In a `return` statement, the parentheses are unnecessary and can be removed.